aptos-move/framework/aptos-framework/sources/fungible_asset.move

```diff
@@ -466,6 +466,16 @@ module aptos_framework::fungible_asset {
         deposit_with_ref(transfer_ref, to, fa);
     }
 
+    /// Create a fungible asset with zero amount.
+    /// This can be useful when starting a series of computations where the initial value is 0.
+    public fun zero<T: key>(metadata: Object<T>): FungibleAsset {
+        let metadata = object::convert<T, Metadata>(metadata);
+        FungibleAsset {
+            metadata,
```

Comment:
then you could just call object::convert here...

Reply:
Updated